app/Http/Controllers/CommentController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class CommentController extends Controller
+{
+    public function store(Request $request)
+    {
+        $post_id = $this->currentUser()->posts->find($request->post_id)->id;
```

Comment:
Theo logic thì mình có quyền comment vào post của người khác, miễn là post đó public.

```suggestion
        $post = Post::public()->find($request->post_id); // kiểm tra lại cú pháp giúp a chỗ này
```